grakn-kb/src/main/java/ai/grakn/kb/internal/concept/ThingImpl.java

```diff
@@ -275,6 +275,8 @@ private void type(TypeImpl type) {
             //noinspection unchecked
             cachedType.set((V) type); //We cache the type early because it turns out we use it EVERY time. So this prevents many db reads
             type.currentShard().link(this);
+            //noinspection unchecked
+            cachedType.set((V) type);
```

Comment:
Tiny merge error. These two links can be deleted because they are duplicates. Look above.